script.js

```diff
@@ -0,0 +1,117 @@
+let storedNotes = JSON.parse(localStorage.getItem("notes"));
+
+let notes = storedNotes ? storedNotes : [];
+let list = document.getElementById("list");
+let upd = 0,
+  targ = "";
+
+document.getElementById("add-btn").addEventListener("click", function () {
```

Comment:
if the user press `space` as an input, it will add an empty list item. you should handle this case too.